runtime/lua/vim/filetype.lua

```diff
@@ -679,6 +697,16 @@ local extension = {
   bbl = "tex",
   latex = "tex",
   sty = "tex",
+  cls = function(path, bufnr)
+    local line = getline(bufnr, 1)
+    if line[1] == "%" then
```

Comment:
You cannot index strings in Lua, instead use `string.sub`

```suggestion
    if line:sub(1, 1) == "%" then
```

Reply:
That was my bad! I was in a bit of a hurry with the "seed PR"...
(and same below)